### ISSUE TITLE
Training with smaller batch size or lesser gpus

### ISSUE THREAD

**Issue by Eashwar93** [HorizonRobotics/Sparse4D#109]
Hi @linxuewu @hlnull ,

First of all thanks for the great work. 

I tried recreating your baseline results with resnet50 backbone trained on 8 GPUs with batch size of 6/GPU resulting in an effective batch size of 48. I was able to reproduce the results with ~46mAP. 

However, when I tried to start a training with 4 GPUs with a batch size of 6/GPU with an effective batch size of 24, the training diverges resulting in ~0 mAP. Did you also observe these kinds of training stability issues? Is the higher batch size always required to stabilize the trainings.

Any comments or insights regarding this would be helpful to understand the issue better :)

**Comment by dlwsds** [comment]
@Eashwar93 I'm experiencing the same issue. Have you managed to resolve it? Could you share any advice? Thanks!

**Comment by Eashwar93** [comment]
Hey @dlwsds ,

I added more gradient clipping, currently the authors use 25 and I ended up using 5. My training doesn't diverge anymore. You can give it a shot but I'm not sure if this is the right fix. Here is the [link](https://github.com/HorizonRobotics/Sparse4D/blob/main/projects%2Fconfigs%2Fsparse4dv3_temporal_r50_1x8_bs6_256x704.py#L431) to the parameter

**Comment by dlwsds** [comment]
> Hey [@dlwsds](https://github.com/dlwsds) ,
> 
> I added more gradient clipping, currently the authors use 25 and I ended up using 5. My training doesn't diverge anymore. You can give it a shot but I'm not sure if this is the right fix. Here is the [link](https://github.com/HorizonRobotics/Sparse4D/blob/main/projects%2Fconfigs%2Fsparse4dv3_temporal_r50_1x8_bs6_256x704.py#L431) to the parameter

Thank you so much for your previous response!  I hope it's okay to ask a follow-up question:  

When I modify the configuration in 'sparse4dv3_temporal_r50_1x8_bs6_256x704.py' with:  
```
total_batch_size = 48
num_gpus = 6
```  
and update the GPU settings in 'local_train.sh' to:  
```
export CUDA_VISIBLE_DEVICES=0,1,2,3,4,5
```  
the training fails with the following error:  
```
assert self.groups_num >= self.global_batch_size
```  
self.groups_num: 16
self.global_batch_size: 48

Would you kindly advise if adjusting 'sequences_split_num' in the configuration might resolve this issue?
